.github/workflows/build.yml

```diff
@@ -75,6 +75,11 @@ jobs:
       - name: "Install: Rust toolchain"
         uses: dsherret/rust-toolchain-file@v1
 
+      - name: "Install: Foundry"
```

Comment:
it would be nice to check solidity code on CI (see #4059)

Reply:
Add it later